vhdbuilder/packer/install-dependencies.sh

```diff
@@ -295,9 +297,14 @@ for imageToBePulled in ${ContainerImages[*]}; do
 
   for version in ${versions}; do
     CONTAINER_IMAGE=$(string_replace $downloadURL $version)
-    pullContainerImage ${cliTool} ${CONTAINER_IMAGE}
+    pullContainerImage ${cliTool} ${CONTAINER_IMAGE} &
+    containerImagePids+=($!)
     echo "  - ${CONTAINER_IMAGE}" >> ${VHD_LOGS_FILEPATH}
+    while [[ $(jobs -p | wc -l) -ge 13 ]]; do # Set maximum number of parallel pulls to 13
```

Comment:
Can you add the data you used for determine 13 as the limit for parallel pulls?

Reply:
And move this to a separate variable?

Reply:
This needs to be explainable to someone who sees this in the future who doesn't have the context we do

Reply:
Is it possible to calculate/estimate 13 based on the VM spec? Let's say we switch to a different VM size in the build pipeline, it'll be nice to adjust this dynamically,